packages/babel/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@rollup/plugin-babel",
+  "version": "5.0.0-alpha.1",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Seamless integration between Rollup and Babel.",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "author": "Rich Harris",
+  "homepage": "https://github.com/rollup/plugins/tree/master/packages/babel",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "./dist/rollup-plugin-babel.cjs.js",
```

Comment:
```suggestion
  "main": "./dist/index.js",
```

Reply:
do we have to do this? IMHO this makes stack traces less readable and using file search in dev tools a little bit less helpful, would really prefer keeping a specific file name here and would like to advocate to change this in all plugins

Reply:
yes, please.

IMHO it makes a stack trace _less_ readable. In a stack trace there is no need for redundancy.

```
.../node_modules/@rollup/plugin-babel/dist/index.js
```

is far cleaner than

```
.../node_modules/@rollup/plugin-babel/dist/rollup-plugin-babel.cjs.js
```

Captain Redundancy Captain does not work here :smile:

Reply:
I can ofc tweak this to whatever you like - but personally I would still vote for a meaningful file name there. Take a look at an example screenshot from my current debugging session:
<img width="330" alt="Screenshot 2020-01-08 at 22 41 19" src="https://user-images.githubusercontent.com/9800850/72018623-59a20780-3268-11ea-8bc8-f64857fdb426.png">
All of those `index.js` are totally unhelpful for me until I click through every one of them.

Reply:
I'll agree, in that context that's a compelling argument. I'm also good with changing the convention across the repo as long as it's consistent. With that said, the one thing I absolutely abhor is incredibly long variable names and incredibly long filenames. 

I propose `dist/@rollup.babel.js` and  `dist/@rollup.babel.es.js`.

The `@` is a valid file character in Windows according to this: https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions. It's a shame we can't use path separator characters in a Windows filename, but it is what it is. That should make it glaringly obvious what the scope of the file is, and what the context of the file is.

Reply:
@shellscape do we want to change this on master? Should I prepare a PR for this? Or should I just adjust it here to the current convention?